test/extension.multiline.test.ts

```diff
@@ -48,6 +48,32 @@ describe('Multiline template tests', () => {
       | \t.b        >> \t.b
       | \t.c++{let} >> \t.c++`)
 
+  Test(`return template - method call (equal indentation)
+      | \tobject.call() >> \treturn object.call()
+      | .anotherCall()  >> \t.anotherCall()
+      | .lastOne(){return} >> \t.lastOne()`)
+
+  Test(`return template - method call (indentation - tabs)
+      | \t\tobject.call()     >> \t\treturn object.call()
+      | \t\t\t.anotherCall()  >> \t\t\t.anotherCall()
+      | \t\t\t.lastOne(){return} >> \t\t\t.lastOne()`)
+
+  // first line gets to keep original indentation in VSCode
+  Test(`return template - method call (indentation - spaces)
+      | ${indent(2)}object.call()   >> ${indent(2)}return object.call()
+      | ${indent(3)}.anotherCall()  >> \t\t\t.anotherCall()
+      | ${indent(3)}.lastOne(){return} >> \t\t\t.lastOne()`)
+
+  Test(`return template - method call (indentation - mixed)
+      | \t\tobject.call()          >> \t\treturn object.call()
+      | ${indent(3)}.anotherCall() >> \t\t\t.anotherCall()
+      | \t\t\t.lastOne(){return}      >> \t\t\t.lastOne()`)
+
+  Test(`return template - method call (indentation - compreturnely mixed)
```

Comment:
`compreturnely`? :)

Reply:
:D good catch, I was obviously using multicursor editing on these tests, though have no idea how I did it with the name